.github/workflows/functional-test-cloud.yaml

```diff
@@ -827,6 +827,11 @@ jobs:
           echo "Pod logs saved to recipes/pod-logs/"
           # Get kubernetes events and save to file
           kubectl get events -n $namespace > recipes/pod-logs/events.txt
+      
+      - name: TEMP list all namespaces on the cluster
+        run: |
+          echo "Listing all namespaces in the cluster:"
+          kubectl get namespaces
```

Comment:
I'll remove these after the PR is reviewed - I'm leaving them in for now so that the reviewer can go check the CI and see that all of these resources are actually being deleted.